UKHO.ADM/internal/ConvertTo-GroupObjects.ps1

```diff
@@ -20,6 +20,16 @@ function ConvertTo-GroupObjects {
                 $ggGroup = [ADGroup]::new($Settings.GroupPrefix, $Settings.Environment, $OU.Name, $GroupConfig.Name, "GG", $OU.DistinguishedName, $OU.Domain)
                 $dlGroup.ADGroupMembers += $ggGroup
 
+                # Add Other Groups
+                if ($null -ne $GroupConfig.Groups -and $GroupConfig.Groups.Count -gt 0) {
+                    $GroupConfig.Groups | ForEach-Object {
+                        $groupGroup = [ADGroup]::new($_.DistinguishedName, $OU.Domain)
+                        if (Check-GroupExists $groupGroup) {
+                            $dlGroup.ADGroupMembers += $groupGroup
```

Comment:
From the users perspective, I don't know if this "Check-GroupExists" is useful. If the user has made a typo this will silently not add the group which could be very annoying as it appear that ADM is just ignoring that group.

If we do check for the groups existence here, can we also log that the group doesn't exist so won't be added? 

The similar scenario is `Check-UserExists` which is invoked in [AddUserToG](https://github.com/UKHO/ADM-PSmodule/blob/328e6d726700409a66390cb6b7de1f6f9c5522bd/UKHO.ADM/Objects/Objects.ps1#L203) so you might be able to copy that existing pattern. But that would only throw the error at "invoke" time which would be annoying.

Reply:
doing it, written to Write-Verbose pipeline